### ISSUE TITLE
Debug comma: place it right before text, rather than at the beginning of line

### ISSUE THREAD

**Issue by zenzeinet** [DaxStudio/DaxStudio#661]
Hi,
The new Debug Commas feature is great, as many of us like the comma at the beginning  (not only for debugging).
The only drawback is that is currently places the comma at the very first column rather than before the text, which is highly unusual

Could you please add an option to put the comma at the proper indentation (i.e. before the first letter of the next line) rather than at the beginning of the line? Would be greatly appreciated. 

Thanks a lot!

**Comment by dgosbell** [comment]
At this point we do not have not plans to implement a feature like this. The decision to put the commas at the start of the line is unusual, but it was a deliberate one as we wanted to make it clear this was a temporary debugging format  (Marco discusses this here https://www.youtube.com/watch?v=YkCMgyTlvXA )

**Comment by zenzeinet** [comment]
Yes, I know it was deliberate. I actually commented to that video and Marco
suggested I add the request here.

As I told him, I agree with pretty much all of your best practices. But the
only one, that does not affect code quality at all, is formatting styles,
and it would be great to be able to choose this.

All our sql is formatted with commas first, and we would rather use the
same style on Dax. It is only cosmetic, but it matters to us.

So hoping you can change your mind, or put it in the backlog for some
future time.

Thanks for everything and for a great tool!

**Comment by dgosbell** [comment]
OK, I'll re-open this and I've created a new "wishlist" label for capturing some of the ideas on the longer term backlog. There are some larger internal improvements I'd like to make which would potentially make a number of features like this one much simpler to implement.

**Comment by zenzeinet** [comment]
Great, thanks Darren!
